python/sglang/srt/model_loader/weight_utils.py

```diff
@@ -584,6 +456,10 @@ def download_weights_from_hf(
 
         # Only perform validation and retry in CI to avoid overhead for regular users
         if is_in_ci():
```

Comment:
Can we move this logic to the new file as well so that all the CI related logics are in the same file?